src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureInfo, CalculatableArea {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
Not fixed: https://github.com/mate-academy/jv-oop-advanced/pull/1508#discussion_r1551425007